index.js

```diff
@@ -374,6 +384,8 @@ class TagInput extends Component {
                   blurOnSubmit={false}
                   onKeyPress={this.onKeyPress}
                   value={text}
+                  placeholder={this.props.placeholder}
```

Comment:
what would happen here if no placeholder was given?  would it just be undefined and there'd be nothing?